lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -788,6 +616,192 @@ class ReportUIFeatures {
   }
 }
 
+class DropDown {
+  /**
+   * @param {DOM} dom
+   */
+  constructor(dom) {
+    /** @type {DOM} */
+    this._dom = dom;
+    /** @type {HTMLElement} */
+    this.toolsButton; // eslint-disable-line no-unused-expressions
+    /** @type {HTMLElement} */
+    this.toolsDropDown; // eslint-disable-line no-unused-expressions
```

Comment:
there's a bunch of renames we can do now.. basically I think we can avoid using 'dropDown' within the class now that its properly dedicated. :)

these sound good to me and @brendankenny : 

```
this.toolsButton => this.toggleEl
this.toolsDropDown => this.menuEl

onToolsButtonClick => onToggleClick, etc
onToolsDropDownKeydown => onMenuKeydown
_getNextDropDownItem => _getNextMenuItem, etc

setupToolsDropDown => setup
closeToolsDropDown => close
openToolsDropDown => open
```

Reply:
Thanks Paul, I will get these renamed tomorrow!

Reply:
Done!